src/platforms/java/common/performance/index.mdx

```diff
@@ -166,3 +166,50 @@ if (span != null) {
     }
 }
 ```
+
+### Connecting Errors with Spans
+
+Sentry errors can be linked with transactions and spans.
+
+Errors reported to Sentry while transaction or span is running are linked automatically:
+
+```java
+ISpan span = Sentry.startTransaction("task");
+try {
+    processItem();
+} catch (Exception e) {
+    Sentry.captureException(e);
+} finally {
+    span.finish();
+}
```

Comment:
should we split these 2 code snippets into the include folder? they are gonna be reused by Android and right now they are under `platforms/java/common/performance/index.mdx`.

otherwise, every fix should be copied over 2 MDX files